src/v/security/exceptions.h

```diff
@@ -22,9 +22,8 @@ namespace security {
 
 struct exception final : public std::runtime_error {
 public:
-    explicit exception(errc e, std::string_view msg)
-      : std::runtime_error(
-        fmt::format("{}: {}", make_error_code(e).message(), msg)) {}
+    explicit exception(std::error_code e, std::string_view msg)
```

Comment:
question: why do youuse `std::runtime_error` as a base rather than `std::system_error` which accepts `std::error_code`'s.  I've noticed this pattern and was just curious ;)

Reply:
> question: why do youuse `std::runtime_error` as a base rather than `std::system_error` which accepts `std::error_code`'s. I've noticed this pattern and was just curious ;)

Just making minimal changes to `security::exception` which has been around for a while, I could make a new exception type in `security::oidc`.

Reply:
up to you, was just curious :)

Reply:
Kept as-is